src/test/java/com/github/bibsysdev/nva/cristin/person/CristinApiClientStub.java

```diff
@@ -0,0 +1,37 @@
+package com.github.bibsysdev.nva.cristin.person;
+
+import java.io.InputStreamReader;
+import java.net.URL;
+import nva.commons.core.JacocoGenerated;
+
+@JacocoGenerated
+public class CristinApiClientStub extends CristinApiClient {
```

Comment:
This class should be removed. I don't think this how one meant to be doing stubbing, in any case it interferes with test coverage. Using wiremock to mock the cristin api dependency makes more sense imo.